sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ControlBlockEditorServiceTest.java

```diff
@@ -309,15 +312,44 @@ void configureNetworkForAllControlBlocks_should_create_SMV_elements() {
         MarshallerWrapper.assertValidateXmlSchema(scd);
     }
 
+    public static Stream<Arguments> ProvideCommunicationToReuse() {
+        TCommunication communicationWithDeletedCb = new TCommunication();
```

Comment:
never used

Reply:
It is used twice. Look at `@MethodSource` of `ParameterizedTest`.

Reply:
communicationWithDeletedCb has been removed.